boefjes/boefjes/plugins/kat_rpki/main.py

```diff
@@ -13,78 +14,155 @@
 
 from boefjes.job_models import BoefjeMeta
 
+# Paths and URLs for RPKI
 BASE_PATH = Path(getenv("OPENKAT_CACHE_PATH", Path(__file__).parent))
 RPKI_PATH = BASE_PATH / "rpki.json"
 RPKI_META_PATH = BASE_PATH / "rpki-meta.json"
 RPKI_SOURCE_URL = "https://console.rpki-client.org/vrps.json"
+
+# Paths and URLs for BGP
+BGP_PATH = BASE_PATH / "bgp.jsonl"
+BGP_META_PATH = BASE_PATH / "bgp-meta.json"
+BGP_SOURCE_URL = "https://bgp.tools/table.jsonl"
+
+# Cache timeout and default hash function
 RPKI_CACHE_TIMEOUT = 1800  # in seconds
-HASHFUNC = "sha256"
+HASHFUNC = getenv("HASHFUNC", "sha256")
+USER_AGENT = getenv("USERAGENT", default="OpenKAT")
```

Comment:
Moving the call to `getenv` outside `run` to the module scope means `getenv` is called when the boefje is imported instead of when the boefje is run. As far as I can see this will mean that any boefje settings in the katalogus will not be used.

Reply:
Thanks for this. I moved everything in `run`!

Reply:
Oeh, that is a tricky little issue I did not realize, we should check the other boefjes for this behaviour.